spec/system/settings/login_activities_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Login activities page' do
+  let!(:user) { Fabricate :user }
+  let!(:login_activity) { Fabricate :login_activity, user: user }
+
+  context 'when signed in' do
+    before { sign_in user }
+
+    describe 'Viewing the login activities page' do
+      it 'shows the login activity history' do
+        visit edit_user_registration_path
+
+        click_on I18n.t('sessions.view_authentication_history')
+
+        expect(page)
+          .to have_content(browser_description)
```

Comment:
This thing could use some chopping up into spans for better responsive appearance.